docs/apache-airflow/best-practices.rst

```diff
@@ -109,6 +109,9 @@ or if you need to deserialize a json object from the variable :
 
     {{ var.json.<variable_name> }}
 
+Alternative option is to read from the environment variables in the top level python code instead of Airflow variables.
+This will avoid new connections to Airflow metadata DB everytime Airflow parses the python file.
```

Comment:
```suggestion
An alternative option is to use environment variables to manage Airflow Variables. This will avoid new connections to Airflow metadata DB every time Airflow parses the python file. For more information, see: :ref:`managing_variables`.
```

Reply:
From Airflow >= 1.10.10 -- you can use Environment Variables to set Airflow Variables, hence connections to DB are not made.